src/eduid_webapp/idp/__init__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 NORDUnet A/S
```

Comment:
Should probably update the copyright to "Copyright (c) 2019 SUNET".

Reply:
fixed